pkg/apis/integreatly/v1alpha1/rhmiconfig_types.go

```diff
@@ -210,6 +210,13 @@ func (h *rhmiConfigMutatingHandler) Handle(ctx context.Context, request admissio
 		rhmiConfig.Annotations["lastEditTimestamp"] = time.Now().UTC().Format(DateFormat)
 	}
 
+	if rhmiConfig.Spec.Maintenance.ApplyFrom == "" {
+		rhmiConfig.Spec.Maintenance.ApplyFrom = DefaultMaintenanceApplyFrom
+	}
+	if rhmiConfig.Spec.Backup.ApplyOn == "" {
+		rhmiConfig.Spec.Backup.ApplyOn = DefaultBackupApplyOn
+	}
```

Comment:
I just noticed this code change.
Since we are in code freeze, we likely shouldn't be merging it.

This PR looks like testing only PR. @ciaranRoche is this code change required?

Reply:
Ill remove this, and add it to a follow up PR for post code freeze, so we can get this testing in.

Reply:
Sounds good to me, thanks.